dashboard/src/pages/ingest/sips/index.vue

```diff
@@ -204,7 +223,7 @@ watch(
 
     <div class="text-muted mb-3">
       Showing {{ sipStore.page.offset + 1 }} -
-      {{ sipStore.lastResultOnPage }} of
+      {{ sipStore.page.offset + sipStore.page.limit }} of
```

Comment:
This may not be true in the final page, where the amount of records doesn't have to match the limit. Also, it currently shows `1 - 0 of 0` if there are no results, I think now it will show `1 to 20 of 0` in that case.

Reply:
I've added a `ResultCounter` component to show a more readable count of the shown results on the SIPs and AIPs pages.  It now shows:

- "No results found" (0 results)
- "Found 1 result" (1 result)
- "Found 10 results" (more than 1 result, but less than two pages of results)
- "Showing 1 - 20 of 35 results" (two or more pages of results)

Reply:
I've made the `ResultCounter` it's own commit for review, but I think I'll squash it with the the dashboard pager commit before merging.